pkg/dogstatsd/parser.go

```diff
@@ -359,3 +360,11 @@ func parseMetricMessage(message []byte, namespace string, namespaceBlacklist []s
 
 	return sample, nil
 }
+
+func parseFloat64(rawFloat []byte) (float64, error) {
```

Comment:
can you add a comment on this function that explains why we're doing this, and links to related golang issue, etc?